src/blockcerts-verifier/BlockcertsVerifier.js

```diff
@@ -54,7 +54,7 @@ class BlockcertsVerifier extends LitElement {
 }
               <buv-verification-modal></buv-verification-modal>
             </section>
-            <buv-footer interactive></buv-footer>
+            ${!_props.hideFooter ? '<buv-footer interactive></buv-footer>' : ''}
```

Comment:
Regarding this, I didn't see a need for a new one, I just had the `disable-other-verification` flag to turn off the interactive one, since the only thing the interactive does for the footer is present that link and render the input.